### ISSUE TITLE
How to convert parameters from Mask R-CNN for C++ app

### ISSUE THREAD

**Issue by manuelrames** [Kolkir/mlcpp#21]
I saw you converted the model pretrained weights to a suitable format to be imported into the C++ implementation. However, I could not find how you did it on the repo. I would like to export the weights of my own  trained Mask R-CNN on the same format so as to use them in you implementation. Would it be possible to see an example of that, please? Thanks a lot and nice job!

**Comment by Kolkir** [comment]
@manuelrames Hello, please take a look on [stateloader.h file](https://github.com/Kolkir/mlcpp/blob/master/mask_rcnn_pytorch/stateloader.h) to see how to export and load parameters from python to C++. Also you can look at this [forum thread](https://discuss.pytorch.org/t/how-to-copy-network-parameters-in-libtorch-c-api/32221).
